Autopilot/Path Manager/MainBatterySensor.c

```diff
@@ -6,40 +6,44 @@
  */
 #include "main.h"
 #include "../Common/Common.h"
-#include "voltageSensor.h"
+#include "MainBatterySensor.h"
 
 
-char percent = 0;
-int currentVoltage = 0;
+int percent = 0;
```

Comment:
It should never be above 100 in the first place. If it does you've got more issues to worry about